From e0b347f1acbe9b1dc8f25298c84496b49960263e Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Tue, 2 May 2023 15:26:57 +0200
Subject: [PATCH] struct ci_hdrc: hide new member at end
References: git-fixes
Patch-mainline: Never, kABI fixup

We can just hide the new member at the end

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/chipidea/ci.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/chipidea/ci.h b/drivers/usb/chipidea/ci.h
index 1c2393644..e140e3b6a 100644
--- a/drivers/usb/chipidea/ci.h
+++ b/drivers/usb/chipidea/ci.h
@@ -260,7 +260,9 @@ struct ci_hdrc {
 	bool				in_lpm;
 	bool				wakeup_int;
 	enum ci_revision		rev;
+#ifndef __GENKSYMS__
 	struct mutex                    mutex;
+#endif
 };
 
 static inline struct ci_role_driver *ci_role(struct ci_hdrc *ci)
-- 
2.40.1

